pkg/apis/core/validation/validation.go

```diff
@@ -4808,11 +4814,17 @@ func ValidateSecurityContext(sc *core.SecurityContext, fldPath *field.Path) fiel
 	}
 
 	if sc.RunAsUser != nil {
-		if *sc.RunAsUser < 0 {
+		if validation.IsValidUserID(*sc.RunAsUser) != nil {
```

Comment:
Nice catch - you might actually want to consider opening a separate PR for this one.

Reply:
fixed